Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,93 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+joint operational direction of the Node.js Technical Steering Committee (TSC)
+and Node.js Foundation Community Committee (CommComm).
+
+## Node.js Admin Repository
+
+The Node.js Admin Repository (nodejs/admin) will serve the central location for
+managing Node.js GitHub Organization administrative activities. Only Node.js
+GitHub Organization owners will have write permissions to the nodejs/admin
+repository.
+
+## Organization Roles
+
+### Owners
+
+The Node.js TSC Director, TSC Chair, and Community Committee Chair, shall be
+granted Owner permissions within the Node.js GitHub Organization.
+
+Additional TSC, CommComm, and Core Technical Committee (CTC) members may be
+granted Owner permissions by request by opening an issue in the nodejs/admin
+repository. Assuming there are no objections from any current voting TSC or
+CommComm member, such requests are approved automatically after 72 hours. If
```

Comment:
I am a bit lost on why TSC, commcomm and CTC can ask for being owners, but only TSC and commcomm vote.

Maybe it would be better to hold off for this for the merger to complete, so that we can simplify some of the language here?